hknweb/events/views.py

```diff
@@ -4,13 +4,32 @@
 from django.contrib import messages
 from django.shortcuts import get_object_or_404
 from django.views.decorators.csrf import csrf_exempt #doing this for now bc idk how to make csrf work
-from django.db.models import F # Avoids changing database values without risking a race condition
+from django.conf import settings
 from django.contrib.auth.decorators import login_required, permission_required
+import datetime
+import pytz
```

Comment:
Given the below changes to use `from django.utils import timezone`, this would make these two imports (`datetime`, `pytz`) unnecessary.